liboptv/include/multimed.h

```diff
@@ -0,0 +1,73 @@
+#ifndef MULTIMED_H
+#define MULTIMED_H
+
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "calibration.h"
+#include "parameters.h"
+#include "ray_tracing.h"
+#include "trafo.h"
+#include "tracking_frame_buf.h"
+#include "lsqadj.h"
+
+
+typedef struct {
+    double x, y, z;
+} Origin; 
+
+/* mmLUT structure */
+typedef struct {
+    Origin origin;
+    int    nr, nz, rw;
+    double *data; 
+} mmlut;
+
+
+
+double get_mmf_from_mmLUT (int i_cam, double X, double Y, double Z, mmlut *mmLUT);
```

Comment:
Here and in other functions, I propose to make mmLUT the first argument, like 'self' in Python.